careers/careers/wordpress.py

```diff
@@ -98,3 +97,70 @@ def process_excerpt(excerpt):
         summary = summary[:-3] + '…'
 
     return summary
+
+
+def get_featured_post(posts_data):
+    featured_post_data = None
+    featured_post_index = None
+
+    for idx, post in enumerate(posts_data):
+        if 'story' in post['tags']:
+            featured_post_data = post
+            featured_post_index = idx
+            break
+
+    if featured_post_index:
+        del posts_data[featured_post_index]
+
+    return posts_data, featured_post_data
+
+
+def prepare_post_data(post_data):
+    # sometimes a post doesn't have a `post-large` image
+    try:
+        image = post_data['featured_media']['media_details']['sizes']['post-large']['source_url']  # noqa
+    except Exception:
```

Comment:
I always try to keep `Exceptions` as narrow as possible. In this case `KeyError`